profiles/models/organization.py

```diff
@@ -0,0 +1,42 @@
+from audioop import reverse
+
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+from django.urls import reverse
+from profiles.models.scope import Scope
+
+
+class Organization(Scope):
+
+    def __str__(self):
+        return self.name
+
+    def get_scopes(self):
+        from profiles.models.scope import GlobalPermission
+        from profiles.models.scope import Scope
+        squest_scope = GlobalPermission.load()
+        return Scope.objects.filter(id=squest_scope.id) | Scope.objects.filter(id=self.id)
+
+    def get_perspective_users(self):
```

Comment:
get_users?

Why he queryset uses User.object.all() here?

Reply:
because we should be able to add any user to an organization, it's different for Team where we want to be able to add only the users of the organization to which it belongs